File: Day_22/Q88.java
import java.util.*;

class Q88{
    static Scanner sc = new Scanner(System.in);

    //Remove Spaces
    static String remove_spaces(String str){
        String ans = "";

        for(int i = 0;i < str.length();i++){
            if(str.charAt(i) != ' '){
                ans += str.charAt(i);
            }
        }

        return ans;
    }

    public static void main(String args[]){
        String str = sc.nextLine();

        System.out.println(remove_spaces(str));

        sc.close();
    }
}

File: Day_22/Q88.cpp
#include <bits/stdc++.h>
using namespace std;

//Remove Spaces
string remove_spaces(string str){
    string ans = "";

    for(int i = 0;i < str.length();i++){
        if(str[i] != ' '){
            ans += str[i];
        }
    }

    return ans;
}

int main(){
    string str;
    getline(cin,str);

    cout << remove_spaces(str);

    return 0;
}
